### PR TITLE
fix: making a comment through header actions

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- The reference was missing. Tested locally.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### Preview domain
https://fix-making-comment.preview.app.daily.dev

### CODE REVIEW

**Review by capJavert — APPROVED** [review]

**Comment by sshanzel** [comment]
May look small and unnecessary at first, but it is because we haven't refactored the internals of the comment input.

File: packages/shared/src/hooks/post/useMutateComment.ts [hunk 1]
```diff
@@ -152,10 +152,15 @@ export const useMutateComment = ({
     isLoading: isCommenting,
     isSuccess,
   } = useMutation<MutateCommentResult, unknown, SubmitComment>(
-    (variables) =>
-      requestMethod(graphqlUrl, mutation, variables, {
+    (variables) => {
+      if (isCommenting || isSuccess) {
+        return Promise.resolve(null);
+      }
```
Comment (capJavert) [inline packages/shared/src/hooks/post/useMutateComment.ts:158]: Why this change? It makes sense to add isCommenting but returning empty promise after one success feels like user would not be able to do two comments in a row?

File: packages/shared/src/contexts/WriteCommentContext.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { createContext, useContext } from 'react';
+import { UseMutateCommentResult } from '../hooks/post/useMutateComment';
+
+interface WriteCommentContextProp {
+  mutateCommentResult: UseMutateCommentResult;
```
Comment (capJavert) [inline packages/shared/src/contexts/WriteCommentContext.ts:5]: I think `mutateComment` is better and shorter?

File: packages/shared/src/hooks/post/useMutateComment.ts (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/hooks/post/useMutateComment.ts:158]: They should be able to. This is a safeguard to avoid multiple same comments. However we can maybe add this in a different place to be safer.

**Review by capJavert — APPROVED** [review]
Looks good 👍
